restructure quota mgmt docs (#5181)
getsentry/sentry-docs: src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

Spike protection is an organization-level setting, so once it's triggered, it affects all the projects in the organization, regardless of which project triggered the spike.

### Managing a Spike {#-spike-protection-was-activated----what-should-i-do}

If spike protection has been triggered for your account, you'll receive an email notifying you:

![Spike Protection Email](spike-email.jpg)

To manage the current spike and avoid future spikes, we recommend taking the following steps:

- Set up an [on-demand budget](#on-demand-budget) to ensure you have time to adjust your volume in the event of a future spike in errors.
- Set better [rate limits](#6-rate-limiting) on the DSN keys for the projects related to the spike.
- If it's a specific release version that has caused the spike, add the version identifier to the project's [inbound filters](#3-inbound-data-filters) to avoid accepting events from that release.

To review the error events dropped because of spike protection, go to the "Usage Stats" tab of **Stats** for your Sentry org and select "Errors" in the "Category" dropdown.

### When Does Spike Protection Become "Inactive?"

> Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ It does however mean that the next time events are dropped, with respect to alerts it will count as a "reactivation" of spike protection.

Author: ```suggestion
Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ The next time events are dropped, spike protection will be "reactivated".
```

### Which issues are consuming my quota? {#-what-issues-are-consuming-my-quota}

You can set up a query in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:

![Busiest Projects](manage-event-stream-02.png)

Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest issues:

![Busiest Issues](manage-event-stream-10.png)

Once you know where your quota is being used, you can start taking the steps below to control it.

## 1. Spike Protection {#7-spike-protection}

A spike is both a **large** and **temporary** increase in error event volume. Sentry applies a dynamic rate limit to your account designed to protect you from short-term spikes.

Spike protection is enabled for your organization by default, and when it's enabled, Sentry continually monitors for spikes. You can confirm that it's enabled in **Settings > Subscription**. This page can only be accessed by a Billing or Owner member of your Sentry organization.

> **We also include a one-time grace period for all paid accounts.** If an account depletes its event volume and has never previously triggered a grace period, we continue to accept events for the next three days at a maximum rate of 10,000 events per hour. If you choose not to increase your volume, Sentry will stop accepting events for the remainder of the current billing month. However, if you increase your volume, events accepted during the grace period will be counted as consumed volume.

Author: Does line 52 have anything to do with spike protection? It was with this content in the original so I haven't moved it, but it's not clear to me that it has any relation to spike protection. Can someone confirm? If not, then I'm going to move it.

Reviewer: Grace period almost needs its own section. It's not related to Spike Protection however the two questions are often related for customers that experiencing large surges of events that deplete their w

> <!-- does the paragraph above belong with spike protection or is this an entirely separate thing? does spike protection have to be enabled for the grace period to kick in?? -->

Author: ```suggestion
```

Spike protection is an organization-level setting, so once it's triggered, it affects all the projects in the organization, regardless of which project triggered the spike.

### Managing a Spike {#-spike-protection-was-activated----what-should-i-do}

If spike protection has been triggered for your account, you'll receive an email notifying you:

![Spike Protection Email](spike-email.jpg)

To manage the current spike and avoid future spikes, we recommend taking the following steps:

- Set up an [on-demand budget](#on-demand-budget) to ensure you have time to adjust your volume in the event of a future spike in errors.
- Set better [rate limits](#6-rate-limiting) on the DSN keys for the projects related to the spike.
- If it's a specific release version that has caused the spike, add the version identifier to the project's [inbound filters](#3-inbound-data-filters) to avoid accepting events from that release.

To review the error events dropped because of spike protection, go to the "Usage Stats" tab of **Stats** for your Sentry org and select "Errors" in the "Category" dropdown.

### When Does Spike Protection Become "Inactive?"

> Events will not be dropped during any minute in which you don't send more than the hourly limit that Sentry has calculated for you. After 24 hours without any dropped events, spike protection becomes "inactive" again. This means that it is no longer dropping events, but _it does not mean the system has stopped paying attention._ It does however mean that the next time events are dropped, with respect to alerts it will count as a "reactivation" of spike protection.

Author: This last sentence sounds like a penalty of some sort; is it? Or is just a fact?

Reviewer: You could rephrase this to say that the next time spike protection is activated you will be alerted. There are a lot of statements in the negative in this block which makes the content hard to follow:

> Events will be dropped during any minute in which you send more than the hourly limit that Sentry has calculated for you. 

Feels easier to follow.

> <!-- this last sentence sounds like a penalty of some sort; is it? or is just a fact? -->
> 

Author: ```suggestion
```

2. Reserve real-time notifications for errors that actually break your code.
3. Manage your error quota.

Applying the proper filters, SDK configuration, and rate limits is an iterative and on-going process, but these tips will show you how to get the most out of your error events. These tips are ordered from easiest or least time-consuming to most challenging or potentially time-consuming. Options 1-5 are all things you can do in [sentry.io](https://sentry.io), whereas the remaining ones are things you'll do in the SDK.

## Before You Begin: Check Your Quota Usage {#common-workflows-for-managing-your-event-stream}

You can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming events or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.

> This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.

Author: ```suggestion
This page is accessible to all members of your organization, so Owners in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```

### How can I see a breakdown of incoming events? {#-how-can-i-see-a-breakdown-of-incoming-events}

The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:

![Usage Stats page showing errors](usage-stats-errors.png)

### What are my busiest projects? {#-what-are-my-busiest-projects}

The "Project" table in the "Usage Stats" tab of **Stats** breaks down your data by project, so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings page where you can manage its [inbound filters](#3-inbound-data-filters) and [rate limits](#6-rate-limiting).

### Which issues are consuming my quota? {#-what-issues-are-consuming-my-quota}

<Note>

You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.

</Note>

> You can set up [a query](https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count%28%29&name=All+Events&query=event.type%3Aerror&sort=-title&statsPeriod=14d&yAxis=count%28%29) in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:

Reviewer 2: ```suggestion
You can set up [a query](https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count%28%29&name=All+Events&query=event.type%3Aerror&sort=-count&statsPeriod=14d&yAxis=count%28%29) in **Discover** to take a more proactive approach to resolving your busiest issues. When you're building the query, search for `event.type:error` and then set the columns `issue`, `title`, `project`, and `count()`, as shown below:
```

![Busiest Projects](manage-event-stream-02.png)

> Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest issues:

Reviewer 2: ```suggestion
Once the changes are applied, sort the "Results" table by the "count()" column to display your busiest issues:
```
or if you want to match screenshot
```suggestion
Once the changes are applied, sort the "Results" table by the "COUNT()" column to display your busiest issues:
```
